### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by rpadovani**
Add some useful info in the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
+**WARNING: this software is in pre-alpha. The happy flow works, but not all the flags combinations have been tested.**
+
+**BE CAREFUL USING IT WITH ALREADY EXISTING EBS VOLUME, DATA CAN BE LOST**
```
Comment (alessandrobessi) [inline README.md]: volumeS

File: README.md [hunk 2]
```diff
@@ -1,2 +1,55 @@
...
+
+Please test it and report issues, patches are more than welcome.
+
 # ebs-cargo
-A small utility to simplify loading GBs of data in a EBS volume on AWS
+
+A small utility to simplify loading GBs of data in an EBS volume on AWS.
+
+When you want to load data on a new EBS volume, lot of operations have to be done: logging in AWS, creating a volume, creating an instance, attaching the volume, formatting it, mounting it, download instance keys, using them to sync data, detaching the EBS and so on.
```
Comment (alessandrobessi) [inline README.md]: a lot or lots

Comment (alessandrobessi) [reply README.md]: downloading

File: README.md [hunk 3]
```diff
@@ -1,2 +1,55 @@
...
+
+Wouldn't be simple just having to launch a command indicating which local directory you want to upload?
```
Comment (alessandrobessi) [inline README.md]: simple -> easier?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+
+The simplest usage is launching the main.py indicating which directory you want to sync:
+
+```
+python3 main.py ~/clone-this
+```
+
+Sane defaults are set for all the options.
+
+You can see all the possible options using:
```
Comment (alessandrobessi) [inline README.md]: possible -> available?

**Review by alessandrobessi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lucagrementieri) [inline README.md]: What happens if you run it on an already existing volume?
Maybe you should specify it. I think the user expects that the directory will be copied on the existing volume.

**Review by lucagrementieri — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rpadovani) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (rpadovani) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (rpadovani) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (rpadovani) [reply README.md]: Done

**Review by alessandrobessi — APPROVED** [review]

**Review by lucagrementieri — APPROVED** [review]

**Pull request merged**